build/test/ProjectsToTest.props

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Layout" ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemGroup>
+    <ProjectsToTest Include="ArgumentForwardingTests"/>
+    <ProjectsToTest Include="crossgen.Tests"/>
+    <ProjectsToTest Include="EndToEnd"/>
+    <ProjectsToTest Include="dotnet.Tests"/>
+    <ProjectsToTest Include="dotnet-build.Tests"/>
+    <ProjectsToTest Include="dotnet-build3.Tests"/>
+    <ProjectsToTest Include="dotnet-compile.Tests"/>
+    <ProjectsToTest Include="dotnet-compile.UnitTests"/>
+    <!-- TODO: https://github.com/dotnet/cli/issues/3558 /><ProjectsToTest Include="/ "dotnet-compile-fsc.Tests" />-->
+    <ProjectsToTest Include="dotnet-new.Tests"/>
+    <ProjectsToTest Include="dotnet-pack.Tests"/>
+    <ProjectsToTest Include="dotnet-publish.Tests"/>
+    <ProjectsToTest Include="dotnet-resgen.Tests"/>
+    <ProjectsToTest Include="dotnet-run.Tests"/>
+    <ProjectsToTest Include="dotnet-run.UnitTests"/>
+    <ProjectsToTest Include="dotnet-test.Tests"/>
+    <ProjectsToTest Include="dotnet-test.UnitTests"/>
+    <!-- TODO: https://github.com/dotnet/cli/issues/321 /><ProjectsToTest Include="/"Kestrel.Tests" />-->
```

Comment:
Still an issue

---
In reply to: [69724004](https://github.com/dotnet/cli/pull/3779#discussion_r69724004) [](ancestors = 69724004)